guides/authentication.md

```diff
@@ -212,7 +219,113 @@ client.loginWithOAuth({
 });
 ```
 
-## More
+### More
 
 Here is an article describing more in detail about the authentication process:
 https://doc.cognitedata.com/dev/guides/iam/external-application.html#tokens
+
+## Azure AD auth flow
+
+Azure AD auth flow can be used via the same `loginWithOAuth` method of `CogniteClient`,
+but different `options` structure has to be passed. You'll find what options structure has to be provided in examples below.
+Besides, here list of pre-requirements, that need to be in place to use Azure AD authentication flow:
```

Comment:
The following prerequisites must be in place to use Azure AD authentication flow: